src/app/about/about.component.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <div class="about-card">
```

Comment:
fix all issues I commented on your PR allRecipes

Reply:
It's done